yarn.lock

```diff
@@ -1947,41 +1947,46 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@lit-labs/motion@npm:^1.0.2":
-  version: 1.0.2
-  resolution: "@lit-labs/motion@npm:1.0.2"
+"@lit-labs/motion@npm:^1.0.3":
+  version: 1.0.3
+  resolution: "@lit-labs/motion@npm:1.0.3"
   dependencies:
     lit: ^2.0.0
-  checksum: 598e0be22a3f931ec971fa001e863c5a4dd82f572d8d0214211bde1d6403b00e3c8fafa92f30b0c02b7272bc12510ec40060bf2c8ab18151bdb264cf32f0ef71
+  checksum: 126fd10df999e10e669d82e3961a01d750ffa7597f32427ea452ab92a24fbb7070358fb3a6a19147bb76652aa2d452ba3d73fd7fca49e381f42858ed4baf482d
   languageName: node
   linkType: hard
 
-"@lit-labs/virtualizer@0.7.0-pre.2":
-  version: 0.7.0-pre.2
-  resolution: "@lit-labs/virtualizer@npm:0.7.0-pre.2"
-  dependencies:
-    event-target-shim: ^5.0.1
-    lit: ^2.0.0
-    tslib: ^1.10.0
-  checksum: 88d5fffe73213c305160257e4e3a63a93087a90e63ca6640a81d156faacb68a8f84a82b94153b9364aeab3f5de45484acad1f8dcc2673e8464e394672358a3cc
+"@lit-labs/ssr-dom-shim@npm:^1.0.0":
+  version: 1.0.0
+  resolution: "@lit-labs/ssr-dom-shim@npm:1.0.0"
+  checksum: ccba6675ad631c6f7360af4e12e328e88d877a52ec992ae9254a1be9902ef699b7916d851b073803dd91244d0f709de2c0539a7db643889cfe341ff6e1a74eb5
   languageName: node
   linkType: hard
 
-"@lit-labs/virtualizer@patch:@lit-labs/virtualizer@0.7.0-pre.2#./.yarn/patches/@lit-labs/virtualizer/event-target-shim.patch::locator=home-assistant-frontend%40workspace%3A.":
-  version: 0.7.0-pre.2
-  resolution: "@lit-labs/virtualizer@patch:@lit-labs/virtualizer@npm%3A0.7.0-pre.2#./.yarn/patches/@lit-labs/virtualizer/event-target-shim.patch::version=0.7.0-pre.2&hash=b7346d&locator=home-assistant-frontend%40workspace%3A."
+"@lit-labs/virtualizer@npm:^1.0.1":
+  version: 1.0.1
+  resolution: "@lit-labs/virtualizer@npm:1.0.1"
   dependencies:
-    event-target-shim: ^5.0.1
-    lit: ^2.0.0
-    tslib: ^1.10.0
-  checksum: b879a6f571d25c28092c516e0dc648128457a8fa57558733de8f2ba15de88b426106f88bc06543b99a2e13ccc0d1cabb4d1f65a2f51de00c912b0f3d997df61a
+    event-target-shim: ^6.0.2
+    lit: ^2.5.0
+    tslib: ^2.0.3
+  checksum: e20ab1cefb48768e8b757147a4eb97c2dd5db686b452eb1cf3d6c9ea178c23ed55540f57117597599c2b5abfb9266f068c782fdbf6b4286637ff8d2e6fd9ff0a
   languageName: node
   linkType: hard
 
-"@lit/reactive-element@npm:1.2.1":
-  version: 1.2.1
-  resolution: "@lit/reactive-element@npm:1.2.1"
-  checksum: 58f1b62c54b1899180b11cd44009ee91ac35099f9016259d3b9cc2f969fc920ab7e7ec32a816986ce89a27c632e25568542c93fdfed6730c19c850f7db0ba4cf
+"@lit/reactive-element@npm:1.0.0-rc.4":
```

Comment:
I think that we should keep the resolution for `@lit/reactive-element` since it's actually preventing a duplicate dependency. We can remove that resolution once it's no longer necessary.

Reply:
Done via `yarn set resolution`, which I just learned incredibly confusingly only edits the lock file in v3 😕